src/screens/comments/styles.js

```diff
@@ -0,0 +1,32 @@
+import {StyleSheet} from 'react-native';
+
+export const styles = StyleSheet.create({
+  backButton: {
+    padding: 10,
+  },
+  backText: {
+    fontSize: 18,
+  },
+  container: {
```

Comment:
I think container **should be** into the top of Object. Group StyleSheet object for better readability.
I mean group by sense as you did here
```
  backButton: {
    padding: 10,
  },
  backText: {
    fontSize: 18,
  },
```